README.md

```diff
@@ -7,15 +7,25 @@ ChatGPT-Style Web Interface for Ollama 🦙
 ## Features ⭐
 
 - 🖥️ **Intuitive Interface**: Our chat interface takes inspiration from ChatGPT, ensuring a user-friendly experience.
+
 - 📱 **Responsive Design**: Enjoy a seamless experience on both desktop and mobile devices.
+
 - ⚡ **Swift Responsiveness**: Enjoy fast and responsive performance.
+
 - 🚀 **Effortless Setup**: Install seamlessly using Docker for a hassle-free experience.
+
 - 🤖 **Multiple Model Support**: Seamlessly switch between different chat models for diverse interactions.
+
 - 📜 **Chat History**: Effortlessly access and manage your conversation history.
-- 📤📥 **Import/Export Chat History**: Seamlessly move your chat data in and out of the platform.
+
+- 📤📥 **Import/Export Chat History**: Seamlessly move your chat data in and out of the platform.\
+
 - ⚙️ **Fine-Tuned Control with Advanced Parameters**: Gain a deeper level of control by adjusting parameters such as temperature and defining your system prompts to tailor the conversation to your specific preferences and needs.
+
 - 💻 **Code Syntax Highlighting**: Enjoy enhanced code readability with our syntax highlighting feature.
-- 🔗 **External Ollama Server Connection**: Link to the model when Ollama is hosted on a different server via the environment variable -e OLLAMA_ENDPOINT="http://[insert your Ollama address]".
+
+- 🔗 **External Ollama Server Connection**: You can seamlessly connect to an external Ollama server hosted on a different address by setting the environment variable during the Docker build process. Execute the following command to include the Ollama API endpoint in the Docker image: `docker build --build-arg OLLAMA_API_ENDPOINT="http://[Your Ollama URL]/api" -t ollama-webui .`.
```

Comment:
My understanding is that there is no Ollama API, but rather Ollama uses the ChatGPT API, so calling it OLLAMA may be a bit of a misnomer.

That said, I may be wrong, and I could see a future in which OLLAMA begins to add its own features to create a separate API (maybe that's already the case with /api/tags? Or maybe that's how GPT distinguishes between 3, 3.5, 4, etc too?)

Reply:
I'm uncertain about your point regarding Ollama's use of the ChatGPT API. Could you please provide further clarification? Ollama functions locally and relies on API calls to access the downloaded models.

Reply:
My point is that it's like S3. Whether you're using Amazon or Digital Ocean or Minio, the ENVs are always named according to S3 because S3 is a protocol more than it is a specific product.

```sh
S3_REGION=...
S3_BUCKET=...
S3_PREFIX=...
S3_KEY=...
```
It would be _more_ confusing to do something like

```sh
DIGITAL_OCEAN_BUCKET=...
```

Because everyone just knows it as "S3" regardless of who is actually providing the service.

Likewise, the GPT is generic. Lots and lots of tools are being built that either provide as a service or consume as a client the GPT API.

So naming it `OLLAMA_API_ENDPOINT` is actually confusing - it makes it sound like it's NOT compatible with GPT, but actually it is.

So if someone had another client product that works with GPT and they wanted to try it with ollama, it would be much more consistent for it to either not carry the protocol name at all `API_ENDPOINT`, or to carry the generic protocol name `GPT_API_ENDPOINT`.

Reply:
I had seen is some of the ollama or mistral documentation that they use the GPT API, but in looking at https://platform.openai.com/docs/api-reference/fine-tuning, it doesn't appear to be the case from what I can see.

Reply:
I see what you mean but I think you might be misunderstanding what Ollama is offering. FYI, Ollama does **_not_** make any requests to proprietary OpenAI GPT-related APIs and everything is processed on a local machine, hence needing to download LLMs locally beforehand (it can also be operated offline!). Additionally, Ollama WebUI only supports Ollama APIs at the moment, so I believe it's appropriate to use the name "OLLAMA_API_ENDPOINT" for the env variable. If you have any other questions regarding Ollama to help you understand how it works, feel free to ask, I'll do my best to answer to the best of my ability! Thanks for the explanation though!

Reply:
@tjbck your discord handle appears to be my GitHub account name. If you tag it with the @ I automatically get notifications for this whole discussion. Just wanted to make you aware and commenting here was the only option I saw.
Cool project btw and have a nice Sunday guys!

Reply:
Oops, so sorry 😅. Thanks for your understanding. Have a good day as well!

Reply:
I know it doesn't make requests to OpenAI's GPT, but I was under the impression that it used the same API signatures.

After checking the documentation and API calls, it looks like that was not correct, or I misunderstood.

Agreed, `OLLAMA_*` is the best way to go.